doc/conf.py

```diff
@@ -177,6 +177,7 @@
                  , ('cpp:typeOrConcept', 'std::string_view')
                  , ('cpp:typeOrConcept', 'std::tuple')
                  , ('cpp:typeOrConcept', 'V') # template arguments which should be matched but aren't.
+                 , ('cpp:typeOrConcept', 'T') # template arguments which should be matched but aren't.
```

Comment:
With sphinx v1.7.5, I was able to build the `html` target with `nitpicky = False` and no the `nitpicky_ignore` list.

Reply:
These will be removed as suggested with #3925 .